Fix the installation of pyenchant (#6462)
PyCQA/pylint: doc/whatsnew/2.14.rst

Other Changes
=============

* Pylint can now be installed with an extra-require called ``spelling`` (``pip install pylint[spelling]``).

>   This will requires a local installation of ``pyenchant``. You will still need to install the

Reviewer: ```suggestion
  This will add ``pyenchant`` to pylint's dependencies. You will still need to install the
```
